src/commands/deploy.ts

```diff
@@ -69,15 +78,21 @@ export const createDeployment = async (
             const BUILD_FILE_BLOCK_LIST = ['**/*.*.map'];
             const buildFilesToIgnore = BUILD_FILE_BLOCK_LIST.map((path) => join(projectPath, path));
 
+            const SOURCE_FILES_REQUIRED_LIST = ['.prettierrc', '.eslintrc.js', 'tsconfig.json'];
```

Comment:
Prettier config can be defined as followed:
```
    A "prettier" key in your package.json file.
    A .prettierrc file written in JSON or YAML.
    A .prettierrc.json, .prettierrc.yml, .prettierrc.yaml, or .prettierrc.json5 file.
    A .prettierrc.js, .prettierrc.cjs, prettier.config.js, or prettier.config.cjs file that exports an object using module.exports.
    A .prettierrc.toml file.
```
And it is not a viable solution if the dev want to push the code without prettier (I don't use prettier on personal project, just eslint), some others just don't use anything and we should most likely respect that

Reply:
@SamuelAlev Interesting. In this instance we should remove prettier here, but the regex of allowed files would need to be extended to allow for .json5, .cjs, .yaml and .toml files too. If the user modifies their eslint rules but we don't upload the prettier file that they use it could cause linting errors on our end when performing the review